lib/meta_logger/slicer/default_impl.ex

```diff
@@ -0,0 +1,45 @@
+defmodule MetaLogger.Slicer.DefaultImpl do
+  @moduledoc """
+  Responsible for slicing log entries according to the given max length option.
```

Comment:
This module implements the Slicer behaviour and is the default implementation. This maintains backwards compatibility.